bin/_main

```diff
@@ -232,6 +232,21 @@ elif [ "${1}" == "postprocess" ]; then
     shift
     base_run_dir=$(echo "$@" | sed -e "s/--base-run-dir\(\s\+\|=\)//")
     post_process_run "${base_run_dir}"
+elif [ "${1}" == "reindex" ]; then
+    shift
+    start_es
+    delete_es_cmd="${CRUCIBLE_HOME}/subprojects/core/CommonDataModel/templates/delete.sh"
+    init_es_cmd="${CRUCIBLE_HOME}/subprojects/core/CommonDataModel/templates/init.sh"
+    $podman_run --name crucible-es-delete "${container_common_args[@]}" "${container_rs_args[@]}" $CRUCIBLE_CONTAINER_IMAGE $delete_es_cmd
+    $podman_run --name crucible-es-init "${container_common_args[@]}" "${container_rs_args[@]}" $CRUCIBLE_CONTAINER_IMAGE $init_es_cmd
```

Comment:
We should probably get in the habit of including unique identifiers in pod names so that we don't run into an issue where they conflict if invoked multiple times...of course in this particular case since you are doing a global action maybe it should conflict -- but if that is the case then we should probably check a return code and bail out if it fails to launch.

Reply:
Agreed